src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {};
```

Comment:
try to use more meaningful naming
```suggestion
  const calculator = {};
```